packages/contracts/contracts/discovery/pools/Pool.sol

```diff
@@ -85,17 +138,40 @@ abstract contract Pool is IPool, RisingTide {
         override(IPool)
         returns (uint256 amount)
     {
-        revert("not yet implemented");
+        return investors[_to].uncappedAllocation;
     }
 
     /// @inheritdoc IPool
     function allocation(address _to)
-        external
+        public
         view
         override(IPool)
         returns (uint256 amount)
     {
-        revert("not yet implemented");
+        return _applyCap(investors[_to].uncappedAllocation);
+    }
+
+    //
+    // Internal API
+    //
+
+    /**
+     * Applies the individual cap to the given amount
+     *
+     * @param _amount amount to apply cap to
+     * @return capped amount
+     */
+    function _applyCap(uint256 _amount) internal view returns (uint256) {
+        /// TODO: Should this be in RisingTide?
+        if (!risingTide_isValidCap()) {
+            return 0;
+        }
+
+        if (_amount >= individualCap) {
+            return individualCap;
+        }
+
+        return _amount;
     }
```

Comment:
could make sense to move this function to `RisingTide` entirely. what do you think?